src/trace-plugin-interface.js

```diff
@@ -199,8 +203,9 @@ PluginAPI.prototype.createChildSpan = function(options) {
       options.skipFrames ? options.skipFrames + 1 : 1);
     return new ChildSpan(this.agent_, childContext);
   } else {
-    this.logger_.warn(options.name + ': Attempted to create child span ' +
-      'without root');
+    var pluginName = options.pluginName || 'unknown plugin';
```

Comment:
Actually `pluginName` seems fine, and the default could just be 'trace agent' or nothing at all.